pkg/host/host.go

```diff
@@ -4,13 +4,15 @@
 package host
 
 import (
+	"github.com/google/syzkaller/pkg/log"
 	"github.com/google/syzkaller/prog"
 )
 
 // DetectSupportedSyscalls returns list on supported and unsupported syscalls on the host.
 // For unsupported syscalls it also returns reason as to why it is unsupported.
 func DetectSupportedSyscalls(target *prog.Target, sandbox string) (
 	map[*prog.Syscall]bool, map[*prog.Syscall]string, error) {
+	log.Logf(3, "detecting supported syscalls")
```

Comment:
We log each syscall at level 1, but this single line on level 3. Usually it's the other way around: the more output, the higher the level. Otherwise one will need to go to level 3 just to see this line.